crypto/crypto/src/hash.rs

```diff
@@ -253,6 +253,17 @@ impl HashValue {
             .map_err(|_| HashValueParseError)
             .map(Self::new)
     }
+
+    /// Create a hash value whose contents are just the given integer. Useful for
+    /// generating basic mock hash values.
+    ///
+    /// Ex: HashValue::from_u64(0x1234) => HashValue([0, .., 0, 0x12, 0x34])
+    pub fn from_u64(v: u64) -> Self {
```

Comment:
Guard with `#[cfg(any(test, feature = "fuzzing"))]` would be nice.

Reply:
Yes this flag is very important to avoid accidental misuse of this public api. Note that the crypto lib, is available as a standalone crate in crates.io.